### ISSUE TITLE
Error/Warning in React Native with latest Beta: Package swr has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports"

### ISSUE THREAD

**Issue by beetlebum** [vercel/swr#1310]
Having just installed the latest version of swr `"swr": "^1.0.0-beta.9"` with React Native `"react-native": "^0.64.2"`, the Metro Bundler shows the following warning:
```warn Package swr has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /Users/.../node_modules/swr/package.json```

Node version v16.5.0

**Comment by huozhi** [comment]
I'm playing react native with expo, didn't have see any warinings for it when I installed swr 1.0.0-beta.9.

IMO a library doesn't have to define package.json in exports if it's not necessary. Found a related issue of react-native cli itself [here](https://github.com/react-native-community/cli/issues/1168). Looks like it has some problem with resolving package.json. The issue is more like on react native cli side.
